index.js

```diff
@@ -18,9 +20,17 @@ function markdownitLinkifyImages (md, config) {
       title = ' title="' + md.utils.escapeHtml(token.attrs[token.attrIndex('title')][1]) + '"'
     }
 
+    if (token.attrIndex('width') !== -1) {
```

Comment:
could we do this more generically and collect all tokens?

Reply:
I'm not sure what you mean by _all_ tokens, but I hope this helps: https://github.com/crookedneighbor/markdown-it-linkify-images/pull/43/commits/b7e33298219c45ed9a324ee062a9aabdd52dcb1a